### PR TITLE
Add check for defined peerConnectionConfig before accessing its browserType property

### PR DESCRIPTION
**Pull request by ggarber**
Failing in prod

### CODE REVIEW

**Comment by fippo** [comment]
the JS always prevented that but... LGTM with nit

File: features-connection.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ function determineBrowserFromOLine(sdp) {
 function gatheringTimeTURN(protocol, client, peerConnectionLog) {
     const peerConnectionConfig = getPeerConnectionConfig(peerConnectionLog);
     let typepref;
-    switch(peerConnectionConfig.browserType) {
+    switch(peerConnectionConfig && peerConnectionConfig.browserType) {
```
Comment (fippo) [inline features-connection.js]: that is an odd switch. Can you just return above if !peerConnectionConfig?

**Comment by fippo** [comment]
wait, getPeerConnectionConfig has a default value already (see features-connection.js at the top). Do you have the dump associated with that?

**Comment by ggarber** [comment]
@fippo Just create a ws, send a random json (f.e. "{}")  and close the connection.   

The getPeerConnectionConfig returns undefined in "create" log is not found.   I can change it in getPeerConnectionConfig if you want, but in rest of places is done this way.   For example:
```!!(peerConnectionConfig && peerConnectionConfig.iceServers !== undefined)```

**Comment by fippo** [comment]
how would that even go into this code... anyway, the more robust the better. Can you still fix the switch?

**Pull request merged**